languageServer/src/languageHelpers/yamlHelper.ts

```diff
@@ -0,0 +1,220 @@
+import {
+  DocumentSymbol,
+  SymbolKind,
+  Range,
+  Location
+} from 'vscode-languageserver/node'
+import {
+  Document,
+  isNode,
+  isPair,
+  isScalar,
+  Pair,
+  parseDocument,
+  Scalar,
+  Node,
+  visit
+} from 'yaml'
+import { BaseLanguageHelper } from './baseLanguageHelper'
+import * as RegExes from './regexes'
```

Comment:
Since you can still import the regexes individually (ex.: `filePaths`), I think it'd be better to rename the exported constants in `regexes.ts` to identify them as regexes and simply import them individually. The other solution would be to wrap them all in a `regexes` object and export that instead, but I think the first solution is better.

Reply:
Yes, that's what I tried first, but I worried if anyone would be annoyed about typing a prefix every time they create/use the constant.
But now that I'm writing this I realised that typing `Regexes.something` is exactly the same thing and now I'm feeling really dumb.
I'll change it, thanks!